tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/OpenTelemetry/StartSpanIntegration.cs

```diff
@@ -0,0 +1,79 @@
+// <copyright file="StartSpanIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.OpenTelemetry
+{
+    /// <summary>
+    /// Msmq calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyName = "OpenTelemetry.Api",
+        TypeName = "OpenTelemetry.Trace.Tracer",
+        MethodName = "StartSpan",
+        ReturnTypeName = "OpenTelemetry.Trace.TelemetrySpan",
+        ParameterTypeNames = new[] { ClrNames.String, "OpenTelemetry.Trace.SpanKind", "OpenTelemetry.Trace.SpanContext&", "OpenTelemetry.Trace.SpanAttributes", "System.Collections.Generic.IEnumerable`1[OpenTelemetry.Trace.Link]", "System.DateTimeOffset" },
```

Comment:
What's with the trailing `&`? Is it to do with it being passed by reference? I didn't know we had to do that!

```csharp
                               👇
"OpenTelemetry.Trace.SpanContext&"
```

Reply:
Code comment added in a40b82bdf35012f7276118c853d32a863d7174a7